frontend/src/App.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
```

Comment:
Wenn ich es richtig sehe nutzt du useState und useEffect in der App.js nicht, dann brauchst du die eigentlich auch nicht zu importieren.

Reply:
Da hast du recht. Ich hatte es vorher in der App.js genutzt und beim umstrukturieren hier vergessen zu löschen. Danke dir.